### PR TITLE
Added multilanguage README

### PR DESCRIPTION
**Pull request by syrusakbary**
# Description

Added multilanguage README:
* [x] [English](https://github.com/wasmerio/wasmer/blob/multilang/README.md) (base, no change)
* [ ]  [Chinese](https://github.com/wasmerio/wasmer/tree/multilang/docs/cn) (translation done, will review apart)
* [x]  [Spanish](https://github.com/wasmerio/wasmer/tree/multilang/docs/es) - (translation done, reviewed by @syrusakbary)
* [x]  [French](https://github.com/wasmerio/wasmer/tree/multilang/docs/fr) - (translation done, @Hywan @jubianchi please review)

# Review

- [ ] Add a short description of the the change to the CHANGELOG.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,66 +105,66 @@ qjs >
 
 [👋 Missing a language?](https://github.com/wasmerio/wasmer/issues/new?assignees=&labels=%F0%9F%8E%89+enhancement&template=---feature-request.md&title=)
 
-[rust logo]: ./assets/languages/rust.svg
+[rust logo]: https://raw.githubusercontent.com/wasmerio/wasmer/master/assets/languages/rust.svg
```
Comment (syrusakbary) [inline README.md:108]: Using absolute urls are better since that way the READMEs will render properly anywhere

File: docs/fr/README.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+## Fonctionnalités
+
+* **Rapide et sûr**. Wasmer exécute WebAssembly à une vitesse *quasi native* dans un environnement entièrement bac à sable.
+
+* **Enfichable**. Wasmer prend en charge différents frameworks de compilation pour répondre au mieux à vos besoins (LLVM, Cranelift ...).
+
+* **Universal**. Vous pouvez exécuter Wasmer sur n'importe quelle *plate-forme* (macOS, Linux et Windows) et *chipset*.
+
+* **Conforme aux normes**. Le runtime réussit [test officiel de WebAssembly suite](https://github.com/WebAssembly/testsuite) prenant en charge [WASI](https://github.com/WebAssembly/WASI) et [Emscripten](https://emscripten.org/)
```
Comment (jubianchi) [inline docs/fr/README.md:42]:
```suggestion
* **Conforme aux normes**. Wasmer passe [la suite de tests officielle de WebAssembly](https://github.com/WebAssembly/testsuite) prenant en charge [WASI](https://github.com/WebAssembly/WASI) et [Emscripten](https://emscripten.org/)
```

File: docs/fr/README.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+## Quickstart
+
+Wasmer est livré sans dépendances. Vous pouvez l'installer à l'aide des programmes d'installation ci-dessous:
```
Comment (jubianchi) [inline docs/fr/README.md:46]:
```suggestion
Wasmer est livré sans aucune dépendance. Vous pouvez l'installer à l'aide des programmes d'installation ci-dessous:
```

File: docs/fr/README.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+```bash
+$ wasmer qjs.wasm
+QuickJS - Type "\h" for help
+qjs >
+```
+
+#### Voici ce que vous pouvez faire ensuite:
+
+- [Utilisez Wasmer depuis votre application Rust](https://docs.wasmer.io/integrations/rust)
+- [Publier un package Wasm sur WAPM](https://docs.wasmer.io/ecosystem/wapm/publishing-your-package)
```
Comment (jubianchi) [inline docs/fr/README.md:81]:
```suggestion
- [Publier un paquet Wasm sur WAPM](https://docs.wasmer.io/ecosystem/wapm/publishing-your-package)
```

File: docs/fr/README.md [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+- [En savoir plus sur Wasmer](https://medium.com/wasmer/)
+
+## Intégrations
+
+📦 Le runtime Wasmer peut être utilisé comme une bibliothèque **intégrée dans différentes langues**, vous pouvez donc utiliser WebAssembly _anywhere_.
```
Comment (jubianchi) [inline docs/fr/README.md:86]:
```suggestion
📦  Wasmer peut être utilisé comme une bibliothèque **intégrée dans différents langages**, vous pouvez donc utiliser WebAssembly _n'import où_.
```

File: docs/fr/README.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+| ![Go logo] | [**Go**][Go integration] | [`wasmer` Go package] | [Docs][go docs] |
+| ![PHP logo] | [**PHP**][PHP integration] | [`wasm` PECL package] | [Docs][php docs] |
+| ![Ruby logo] | [**Ruby**][Ruby integration] | [`wasmer` Ruby Gem] | [Docs][ruby docs] |
+| ![Java logo] | [**Java**][Java integration] | [`wasmer/wasmer-jni` Bintray package] | [Docs][java docs] |
+| ![Elixir logo] | [**Elixir**][Elixir integration] | [`wasmex` hex package] | [Docs][elixir docs] |
+| ![R logo] | [**R**][R integration] | *no published package* | [Docs][r docs] |
+| ![Postgres logo] | [**Postgres**][Postgres integration] | *no published package* | [Docs][postgres docs] |
+|  | [**Swift**][Swift integration] | *no published package* | |
+
+[👋 Missing a language?](https://github.com/wasmerio/wasmer/issues/new?assignees=&labels=%F0%9F%8E%89+enhancement&template=---feature-request.md&title=)
```
Comment (jubianchi) [inline docs/fr/README.md:105]:
```suggestion
[👋  Il manque un langage ?](https://github.com/wasmerio/wasmer/issues/new?assignees=&labels=%F0%9F%8E%89+enhancement&template=---feature-request.md&title=)
```

File: docs/fr/README.md [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+[postgres integration]: https://github.com/wasmerio/wasmer-postgres
+[postgres docs]: https://github.com/wasmerio/wasmer-postgres#usage--documentation
+
+[swift integration]: https://github.com/AlwaysRightInstitute/SwiftyWasmer
+
+## Contribuer
+
+**Nous accueillons toute forme de contribution, en particulier de la part des nouveaux membres de notre communauté** 💜
+
+Vous pouvez vérifier comment construire le runtime Wasmer dans [nos super documents](https://docs.wasmer.io/ecosystem/wasmer/building-from-source)!
```
Comment (jubianchi) [inline docs/fr/README.md:176]:
```suggestion
Vous pouvez vérifier comment compiler Wasmer dans [nos super documents](https://docs.wasmer.io/ecosystem/wasmer/building-from-source)!
```

File: docs/fr/README.md [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+
+### Test
+
+Test que vous voulez? La [documentation Wasmer vous montrera comment](https://docs.wasmer.io/ecosystem/wasmer/building-from-source/testing).
```
Comment (jubianchi) [inline docs/fr/README.md:180]:
```suggestion
Vous voulez des tests ? La [documentation de Wasmer](https://docs.wasmer.io/ecosystem/wasmer/building-from-source/testing) vous montrera comment les exécuter.
```

File: docs/cn/README.md [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+| ![Go logo] | [**Go**][Go integration] | [`wasmer` Go package] | [文件资料][go docs] |
+| ![PHP logo] | [**PHP**][PHP integration] | [`wasm` PECL package] | [文件资料][php docs] |
+| ![Ruby logo] | [**Ruby**][Ruby integration] | [`wasmer` Ruby Gem] | [文件资料][ruby docs] |
+| ![Java logo] | [**Java**][Java integration] | [`wasmer/wasmer-jni` Bintray package] | [文件资料][java docs] |
+| ![Elixir logo] | [**Elixir**][Elixir integration] | [`wasmex` hex package] | [文件资料][elixir docs] |
+| ![R logo] | [**R**][R integration] | *没有已发布的软件包* | [文件资料][r docs] |
+| ![Postgres logo] | [**Postgres**][Postgres integration] | *没有已发布的软件包* | [文件资料][postgres docs] |
+|  | [**Swift**][Swift integration] | *没有已发布的软件包* | |
+
+[👋 Missing a language?](https://github.com/wasmerio/wasmer/issues/new?assignees=&labels=%F0%9F%8E%89+enhancement&template=---feature-request.md&title=)
```
Comment (syrusakbary) [inline docs/cn/README.md:106]:
```suggestion
[👋 缺少语言？](https://github.com/wasmerio/wasmer/issues/new?assignees=&labels=%F0%9F%8E%89+enhancement&template=---feature-request.md&title=)
```

File: docs/es/README.md [hunk 10]
```diff
@@ -0,0 +1,191 @@
...
+| ![Go logo] | [**Go**][Go integration] | [`wasmer` en Go] | [Documentación][go docs] |
+| ![PHP logo] | [**PHP**][PHP integration] | [`wasm` en PECL] | [Documentación][php docs] |
+| ![Ruby logo] | [**Ruby**][Ruby integration] | [`wasmer` en Ruby Gems] | [Documentación][ruby docs] |
+| ![Java logo] | [**Java**][Java integration] | [`wasmer/wasmer-jni` en Bintray] | [Documentación][java docs] |
+| ![Elixir logo] | [**Elixir**][Elixir integration] | [`wasmex` en hex] | [Documentación][elixir docs] |
+| ![R logo] | [**R**][R integration] | *sin paquete publicado* | [Documentación][r docs] |
+| ![Postgres logo] | [**Postgres**][Postgres integration] | *sin paquete publicado* | [Documentación][postgres docs] |
+|  | [**Swift**][Swift integration] | *sin paquete publicado* | |
+
+[👋 Missing a language?](https://github.com/wasmerio/wasmer/issues/new?assignees=&labels=%F0%9F%8E%89+enhancement&template=---feature-request.md&title=)
```
Comment (syrusakbary) [inline docs/es/README.md:105]:
```suggestion
[👋 Falta algún lenguaje?](https://github.com/wasmerio/wasmer/issues/new?assignees=&labels=%F0%9F%8E%89+enhancement&template=---feature-request.md&title=)
```

File: docs/fr/README.md [hunk 11]
```diff
@@ -0,0 +1,191 @@
...
+[postgres integration]: https://github.com/wasmerio/wasmer-postgres
+[postgres docs]: https://github.com/wasmerio/wasmer-postgres#usage--documentation
+
+[swift integration]: https://github.com/AlwaysRightInstitute/SwiftyWasmer
+
+## Contribuer
+
+**Nous accueillons toute forme de contribution, en particulier de la part des nouveaux membres de notre communauté** 💜
+
+Vous pouvez vérifier comment compiler Wasmer dans [nos super documents](https://docs.wasmer.io/ecosystem/wasmer/building-from-source)!
```
Comment (Hywan) [inline docs/fr/README.md:176]:
```suggestion
Vous pouvez vérifier comment compiler Wasmer dans [notre documentation](https://docs.wasmer.io/ecosystem/wasmer/building-from-source)!
```

File: docs/fr/README.md [hunk 12]
```diff
@@ -0,0 +1,191 @@
...
+
+[postgres logo]: https://raw.githubusercontent.com/wasmerio/wasmer/master/assets/languages/postgres.svg
+[postgres integration]: https://github.com/wasmerio/wasmer-postgres
+[postgres docs]: https://github.com/wasmerio/wasmer-postgres#usage--documentation
+
+[swift integration]: https://github.com/AlwaysRightInstitute/SwiftyWasmer
+
+## Contribuer
+
+**Nous accueillons toute forme de contribution, en particulier de la part des nouveaux membres de notre communauté** 💜
```
Comment (Hywan) [inline docs/fr/README.md:174]:
```suggestion
**Nous accueillons toutes formes de contributions, en particulier de la part des nouveaux membres de notre communauté**. 💜
```

File: docs/fr/README.md [hunk 13]
```diff
@@ -0,0 +1,191 @@
...
+
+Vous pouvez vérifier comment compiler Wasmer dans [nos super documents](https://docs.wasmer.io/ecosystem/wasmer/building-from-source)!
+
+### Test
+
+Vous voulez des tests ? La [documentation de Wasmer](https://docs.wasmer.io/ecosystem/wasmer/building-from-source/testing) vous montrera comment les exécuter.
+
+## Communauté
+
+Wasmer a une incroyable communauté de développeurs et de contributeurs. Bienvenue, rejoignez-nous! 👋
```
Comment (Hywan) [inline docs/fr/README.md:184]:
```suggestion
Wasmer a une incroyable communauté de développeurs et de contributeurs. Bienvenue et rejoignez-nous ! 👋
```

File: docs/fr/README.md [hunk 14]
```diff
@@ -0,0 +1,191 @@
...
+
+### Canaux
```
Comment (Hywan) [inline docs/fr/README.md:186]:
```suggestion
### Canaux de communications
```

**Review by Hywan — APPROVED** [review]

**Comment by syrusakbary** [comment]
Merging manually as there are no code changes (and we don't need bors).

We'll try to review the chinese translation once the PR is merged (if you are chinese and you want to help us fix any translation mistake that would be awesome!)

**Pull request merged**